simulate.py

```diff
@@ -15,7 +15,7 @@
     'inverse',
     'peak_load_window', 'flex_window',
     'schedule', 'schedule_foresight',
-    'v2g',
+    'v2g', 'distributed'
```

Comment:
distributed should get its own line. Try to end lines with a comma so you can easily extend the list (makes it easier to see changes in the diff)
I wasn't even aware that inverse and v2g are still valid strategies? I think both can be removed.